bin/list-incomplete-messages

```diff
@@ -23,53 +23,53 @@
 #      placeholder.street
 #
 
-pushd $(git rev-parse --show-toplevel) >/dev/null
+source bin/lib.sh
 
-get_messages() {
-  regex="^[a-z][^=]*"
-  echo "$(grep -o ${regex} $1 | sort)"
+function get_messages() {
+  local REGEX="^[a-z][^=]*"
+  echo "$(grep -o ${REGEX} $1 | sort)"
 }
 
-diff_messages() {
-  if [[ ! -z $2 ]]
-  then
-    echo "Missing translations in $3:";
-    difference="$(diff -b --strip-trailing-cr <(echo "$1") <(echo "$2"))";
-    echo "${difference}" | grep "<" | grep -o "[^<][a-zA-Z.]*" | sed "s/^/    /";
+function diff_messages() {
+  if [[ ! -z "${2}" ]]; then
+    echo "Missing translations in $3:"
```

Comment:
consider adding `>&2` to these error messages to send to stderr

Reply:
switched to using `out::error` which does that